File: HrnciaActividad2PG3/src/Clases/Vehiculo.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */
package Clases;

/**
 *
 * @author luisd
 */
public class Vehiculo {

    private String FechaDEfabricacion;
    private String VIN_chasis;
    private String VINMotor;

    public Vehiculo(String FechaDEfabricacion, String VIN_chasis, String VINMotor) {
        this.FechaDEfabricacion = FechaDEfabricacion;
        this.VIN_chasis = VIN_chasis;
        this.VINMotor = VINMotor;

      
    }

    /**
     * @return the FechaDEfabricacion
     */
    public String getFechaDEfabricacion() {
        return FechaDEfabricacion;
    }

    /**
     * @param FechaDEfabricacion the FechaDEfabricacion to set
     */
    public void setFechaDEfabricacion(String FechaDEfabricacion) {
        this.FechaDEfabricacion = FechaDEfabricacion;
    }

    /**
     * @return the VIN_chasis
     */
    public String getVIN_chasis() {
        return VIN_chasis;
    }

    /**
     * @param VIN_chasis the VIN_chasis to set
     */
    public void setVIN_chasis(String VIN_chasis) {
        this.VIN_chasis = VIN_chasis;
    }

    /**
     * @return the VINMotor
     */
    public String getVINMotor() {
        return VINMotor;
    }

    /**
     * @param VINMotor the VINMotor to set
     */
    public void setVINMotor(String VINMotor) {
        this.VINMotor = VINMotor;
    }
}

File: HerenciaPYTActividad2PG3/Clases/Vehiculo.py
class Vehiculo:
    def __init__(self, FechaDEfabricacion, VIN_chasis, VINMotor):
        self._FechaDEfabricacion = FechaDEfabricacion
        self._VIN_chasis = VIN_chasis
        self._VINMotor = VINMotor

    @property
    def FechaDEfabricacion(self):
        return self._FechaDEfabricacion

    @FechaDEfabricacion.setter
    def FechaDEfabricacion(self, nueva_fecha):
        self._FechaDEfabricacion = 	nueva_fecha

    @property
    def VIN_chasis(self):
        return self._VIN_chasis

    @VIN_chasis.setter
    def VIN_chasis(self, nuevo_vin_chasis):
        self._VIN_chasis = 	nuevo_vin_chasis

    @property
    def VINMotor(self):
        return self._VINMotor

    @VINMotor.setter
    def VINMotor(self, nuevo_vin_motor):
        self._VINMotor = nuevo_vin_motor
